src/org/lwjgl/demo/vulkan/raytracing/HybridMagicaVoxel.java

```diff
@@ -1087,6 +1087,8 @@ private static AllocationAndBuffer createBuffer(int usageFlags, long size, ByteB
             // create the final destination buffer
             LongBuffer pBuffer = stack.mallocLong(1);
             PointerBuffer pAllocation = stack.mallocPointer(1);
+            VmaAllocationInfo pAllocationInfo = VmaAllocationInfo.calloc(stack);
+
```

Comment:
Thanks! I think there is no reason to calloc here. Malloc will also work since the struct is expected to be filled fully by vma.

Reply:
Oh, that's true.
Fix'd.